React/Base/RCTConvert.m

```diff
@@ -1145,22 +1145,49 @@ + (NSPropertyList)NSPropertyList:(id)json
 #if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
 + (NSString*)accessibilityRoleFromTrait:(NSString*)trait
 {
-  // a subset of iOS accessibilityTraits map to macOS accessiblityRoles:
-  if ([trait isEqualToString:@"button"]) {
-    return NSAccessibilityButtonRole;
-  } else if ([trait isEqualToString:@"text"]) {
-    return NSAccessibilityStaticTextRole;
-  } else if ([trait isEqualToString:@"link"]) {
-    return NSAccessibilityLinkRole;
-  } else if ([trait isEqualToString:@"image"]) {
-    return NSAccessibilityImageRole;
-    // a set of RN accessibilityTraits are macOS specific accessiblity roles:
-  }	else if ([trait isEqualToString:@"group"]) {
-    return NSAccessibilityGroupRole;
-  } else if ([trait isEqualToString:@"list"]) {
-    return NSAccessibilityListRole;
+  static NSDictionary<NSString *, NSString *> *traitOrRoleToAccessibilityRole;
```

Comment:
is it necessary to hold this in memory?

Reply:
Seeing as @tom-un changed it in response to my question [here](https://github.com/microsoft/react-native-macos/pull/603#discussion_r490822292), I assume the answer was yes.

If every byte counts, perhaps it might help to not add the `none` key to the list and rely on the fallback? 🤷‍♂️

Reply:
Yes, this is the pattern that every other enum converter in this file is doing.   Most cases this file use the `RCT_ENUM_CONVERTER` macro which expands to the same static dictionary with dispatch_once to init pattern.